docs/specs/moniker.yml

```diff
@@ -2,6 +2,8 @@
 # Apply moniker range setting in the config to markdown file's output path and output content
 inputs:
   docfx.yml: |
+    documentId:
+      siteBasePath: docs
```

Comment:
consider promote `documentId/siteBasePath` to `/baseUrl` in another PR, where `BaseUrl` is `https://{hostName}/{basePath}`